docs/content/users/install/ddev-upgrade.md

```diff
@@ -13,6 +13,7 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
 
     ```bash
     # Upgrade DDEV to the latest version
+    brew tap ddev/ddev
```

Comment:
```suggestion
```

This is not necessary, since `brew install ddev/ddev/ddev` does the tap for you.